docs/MYSQL_WC.md

```diff
@@ -0,0 +1,52 @@
+MySQL Working Copy
+-----------------------
+
+In order to use a [MySQL](https://www.mysql.com/) working copy, you need to have server running MySQL. MySQL 8.0 and later is officially supported by Kart (MySQL 5.6 and later are largely compatible but not officially supported).
+
+### MySQL partitioning
+
+MySQL databases are designed so that they can be used for multiple tasks simultaneously without those tasks interfering with each other. This is achieved by storing data from different dasks in different databases.
+
+* A MySQL server contains one or more named databases, which in turn contain tables. A user connected to the server can query tables in any database they have access-rights to without starting a new connection. Two tables can have the same name, as long as they are in different databases.
+
+MySQL has only a single layer of data separation - the *database*. (Contrast to [PostgreSQL](POSTGIS_WC.md) and [Microsoft SQL Server](SQL_SERVER_WC.md) which have two layers, *database* and *schema*). A Kart MySQL working copy is fine to share a server with any other task, but it expects to be given its own database to manage (just as Kart expects to manage its own GPKG working copy, not share it with other data). Managing the database means that Kart is responsible for initialising that database and importing the data in its initial state, then keeping track of any edits made to that data so that they can be committed. Kart expects that the user will use some other application to modify the data in that database as part of making edits to a Kart working copy. (This differs to other working copy types that only manage a single *schema* within a database).
+
+### MySQL Connection URI
+
+A Kart repository with a MySQL working copy needs to be configured with a `mysql://` connection URI. This URI contains how to connect to the server, and the name of the database that should be managed as a working copy by this Kart repository.
+
+A connection URL would generally have the following format:
+
+`mysql://[user[:password]@][host][:port]`
+
+Since Kart also requires the database to be specified up front, Kart needs a connection URL in the following format:
```

Comment:
skip the general URL format, it's easy to miss that that's the _wrong_ format to use, if you're skimming the docs

```suggestion
Kart needs a connection URL in the following format:
```